### PR TITLE
add more collateral to goku money

### PR DESCRIPTION
**Pull request by BuzzSeagull**
add more collateral to goku money

- MANTA
- STONE
- wUSDM
- GAI

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/goku-money exports TVL: 
        
 
 
    manta                     122.42 k
    
    total                    122.42 k

File: projects/goku-money/index.js [hunk 1]
```diff
@@ -10,41 +11,79 @@ const COLLATERALS = {
...
   for (const [collateral, collateralInfo] of Object.entries(COLLATERALS)) {
     const { activePool, defaultPool } = collateralInfo
     tokensAndOwners.push([collateral, activePool])
     tokensAndOwners.push([collateral, defaultPool])
   }
-  return sumTokens2({ api, tokensAndOwners, })
+  return tokensAndOwners;
+}
+
+function getstableOwnersAndToken() {
```
Comment (g1nt0ki) [inline projects/goku-money/index.js:70]: @BuzzSeagull feel like we had this discussion before, we cant count own stablecoin in stability pool as tvl

**Comment by llamatester** [comment]
The adapter at projects/goku-money exports TVL: 
        
 
 
    manta                     112.62 k
    
    total                    112.62 k

File: projects/goku-money/index.js (hunk 1 above)
Comment (BuzzSeagull) [reply projects/goku-money/index.js]: ok, just pushed a new commit, we removed own stable coin in stability pool.

**Pull request merged**